jsonnet/prometheus.jsonnet

```diff
@@ -269,6 +269,34 @@ local namespacesRole =
         },
       },
 
+    serviceMonitorOpenShiftApiserver:
```

Comment:
nit: we could make this a function, as more and more service monitors are popping up with similar/same settings.